daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/types/Ledger.scala

```diff
@@ -162,7 +164,7 @@ object Ledger {
     committer = committer,
     effectiveAt = effectiveAt,
     roots = enrichedTx.roots.map(ScenarioNodeId(commitPrefix, _)),
-    nodes = enrichedTx.nodes.map {
+    nodes = TreeMap.empty[ScenarioNodeId, Node] ++ enrichedTx.nodes.map {
```

Comment:
Probably `collection.breakOut` works better in this case.

Reply:
thanks, will change it.